pkg/operatorsource/appregistryhelper.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
 	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
-	interface_client "github.com/operator-framework/operator-marketplace/pkg/client"
+	interfaceClient "github.com/operator-framework/operator-marketplace/pkg/client"
```

Comment:
This change should be made in the previous commit which is where you introduced it.